README.md

```diff
@@ -1 +1,49 @@
-# leaderboard
\ No newline at end of file
+# leaderboard![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Leaderboard
+
+An application which shows shows using api
+
+## Live Demo
+
+[Live Demo Link](https://kimutai01.github.io/todo-list/)
+
+![screenshot](./leader.png)
+
+## Built With
+
+- HTML5
+- CSS3
+- Visual Studio Code
+- Git & Github
+- Webpack
+
+## Getting Started
+
+### Prerequisites
+
+Any modern web browser will do.
+
+### Setup
+
+Follow the steps described before.
```

Comment:
- [OPTIONAL ] Great job :adding the README file 👏🏻 Let's make it more derscriptive then, Add a descriptive README file to your project,  A professional README file should have a project title, project description, list of technology used in the project, and instructions to set and run the project up locally, This includes `git clone URL` , `cd project_name`,  `npm install`, `npm start`, etc


- [OPTIONAL] Kindly, it would be nice to add explicit steps for cloning and running your app, as this will be of help to non-technical users. You can put the steps to clone and run the project in a new Getting Started section under the project's description. This includes `git clone URL` , `cd..`,  ` npm install`, `npm start`, etc.